[fix #9] [mozmeao/infra#1104] Clean up oregon-a prod yaml.

jpetto opened the pull request.

jpetto committed:
[fix #9] [mozmeao/infra#1104] Clean up oregon-a prod yaml.

jgmize commented on oregon-a/basket-prod/web-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-prod-v38-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-a/basket-prod/web-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-            secretKeyRef:
-              key: celery-redis-max-connections
-              name: basket-prod-v38-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-a/basket-prod/web-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-            secretKeyRef:
-              key: email-port
-              name: basket-prod-v38-env
+          value: 587
```
```suggestion
          value: "587"
```

jgmize commented on oregon-a/basket-prod/web-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-prod-v38-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-a/basket-prod/web-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-prod-v38-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-a/basket-prod/web-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-prod-v38-env
+          value: 10
```
```suggestion
          value: "10"
```

jgmize commented on oregon-a/basket-prod/web-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-prod-v38-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-a/basket-prod/web-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-              key: conflicts
-              name: basket-prod-v38-env
-        image: 127.0.0.1:5555/basket-prod:v38
+          value: 1
```
```suggestion
          value: "1"
```

jgmize commented on oregon-a/basket-prod/web-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-              name: basket-prod-v38-env
-        image: 127.0.0.1:5555/basket-prod:v38
+          value: 1
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jgmize commented on oregon-a/basket-prod/worker-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-            secretKeyRef:
-              key: celery-num-workers
-              name: basket-prod-v38-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-a/basket-prod/worker-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-            secretKeyRef:
-              key: celery-redis-max-connections
-              name: basket-prod-v38-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-a/basket-prod/worker-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-            secretKeyRef:
-              key: email-port
-              name: basket-prod-v38-env
+          value: 587
```
```suggestion
          value: "587"
```

jgmize commented on oregon-a/basket-prod/worker-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-            secretKeyRef:
-              key: redis-cache-db
-              name: basket-prod-v38-env
+          value: 5
```
```suggestion
          value: "5"
```

jgmize commented on oregon-a/basket-prod/worker-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-            secretKeyRef:
-              key: redis-celery-db
-              name: basket-prod-v38-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-a/basket-prod/worker-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-            secretKeyRef:
-              key: workflow-release-summary
-              name: basket-prod-v38-env
+          value: 10
```
```suggestion
          value: "10"
```

jgmize commented on oregon-a/basket-prod/worker-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-            secretKeyRef:
-              key: wsgi-num-workers
-              name: basket-prod-v38-env
+          value: 4
```
```suggestion
          value: "4"
```

jgmize commented on oregon-a/basket-prod/worker-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-              key: conflicts
-              name: basket-prod-v38-env
-        image: 127.0.0.1:5555/basket-prod:v38
+          value: 1
```
```suggestion
          value: "1"
```

jgmize commented on oregon-a/basket-prod/worker-deploy.yaml:
```diff
@@ -48,376 +37,253 @@ spec:
-              name: basket-prod-v38-env
-        image: 127.0.0.1:5555/basket-prod:v38
+          value: 1
+        image: mozmeao/basket:4d0668d
```
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

jpetto committed:
Review fixes:

- remove superfluous data from ns.yaml
- quote all numeric env var values
- use full image hash
